diff --git a/dist/index.js b/dist/index.js
index 493cdc0b8eae78fb25c5c9164b1b1f0ef50bd08d..a1a6331403e7363f222f989e7b67fad59b013d88 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -64,6 +64,14 @@ const ecConfig = (config) => {
-		useStyleReset: config.useStyleReset,
-		useThemedScrollbars: config.useThemedScrollbars,
-		useThemedSelectionColors: config.useThemedSelectionColors,
-		frames: {
-			showCopyToClipboardButton: false,
-			extractFileNameFromCode: false
+		useStyleReset: config.useStyleReset,
+		useThemedScrollbars: config.useThemedScrollbars,
+		useThemedSelectionColors: config.useThemedSelectionColors,
+		shiki: {
+			transformers: [{
+				name: "complete-hover-syntax-highlighting",
+				preprocess(_code, options) {
+					options.tokenizeTimeLimit = 0;
+				}
+			}]
+		},
+		frames: {
+			showCopyToClipboardButton: false,
+			extractFileNameFromCode: false
